### PR TITLE
Listen to karma as an alias of score

### PR DESCRIPTION
**Pull request by rodrigoaguilera**

### CODE REVIEW

File: src/plusplus.coffee [hunk 1]
```diff
@@ -142,7 +142,7 @@ module.exports = (robot) ->
     reasons = scoreKeeper.reasonsForUser(name)
 
     reasonString = if typeof reasons == 'object' && Object.keys(reasons).length > 0
-                     "#{name} has #{score} points. here are some raisins:" +
+                     "#{name} has #{score} points. here are some reasons:" +
```
Comment (ajacksified) [inline src/plusplus.coffee]: Please use topical pull requests and split the previous change (karma as alias) from this verbiage change.

File: src/plusplus.coffee [hunk 2]
```diff
@@ -127,7 +127,7 @@ module.exports = (robot) ->
                   "Erased points for #{name}"
       msg.send message
 
-  robot.respond /score (for\s)?(.*)/i, (msg) ->
+  robot.respond /(?:score|karma) (for\s)?(.*)/i, (msg) ->
```
Comment (ajacksified) [inline src/plusplus.coffee]: This may work better as a config var, which would give you even more flexibility.

Comment (therealklanni) [inline src/plusplus.coffee]: Agreed. Let's make it a config.

File: src/plusplus.coffee (hunk 1 above)
Comment (rodrigoaguilera) [inline src/plusplus.coffee]: Created new PR

File: src/plusplus.coffee (hunk 2 above)
Comment (rodrigoaguilera) [inline src/plusplus.coffee]: Created new PR

**Comment by rodrigoaguilera** [comment]
Splitted this PR in two new

**Pull request closed without merging**